src/iarc7_motion/iarc_tasks/velocity_task.py

```diff
@@ -133,7 +127,14 @@ def get_desired_command(self):
 
                 self._current_velocity = desired_vel
 
-                return (TaskRunning(), VelocityCommand(velocity))
+                if reset_z:
```

Comment:
Is this supposed to reset the motion plan in the linear motion profile generator, or just change the current timestep?  Because right now it looks like it just changes the current timestep